code_to_gast/helpers/py_helpers.py

```diff
@@ -151,4 +151,10 @@ def name_to_gast(node):
 takes node of type unaryOp and converts it to our generic ast represenations
 """
 def unary_op_to_gast(node):
-    return {"type": "unaryOp", "op": pyop_to_str(node.op), "arg": pr.node_to_gast(node.operand)}
\ No newline at end of file
+    return {"type": "unaryOp", "op": pyop_to_str(node.op), "arg": pr.node_to_gast(node.operand)}
```

Comment:
Nit: Having newlines at the end of files will prevent this diff awkwardness where it looks like the last line is changed when it's just a newline.